### DISCUSSION TITLE
Privacy: Any updates on the status of private accounts?

### DISCUSSION THREAD

**Discussion by mayaarane** [bluesky-social/atproto#2809] [Clarifications and "Why?"] [answered]
I know people who refuse to use BlueSky because it doesn't have a way to privatize their account. I'm trying to keep my account open. I think it's a crucial feature for user security as well. I'm aware of the difficulty, but is there any update on whether the devs are thinking about this or are already working on it?

**Comment by bnewbold** [comment, marked as answer]
We are working to add support for shared, personal, and other non-public content in the protocol. We don't have any update or estimate on when or how that might work out in terms of "private accounts" as a specific feature in the Bluesky Social app (`app.bsky.*` lexicons).

**Reply by mayaarane** [reply]
I hope the private account feature comes someday. It's REALLY essential for user protection and I'm sure you've seen a lot of requests for it. I'm also looking forward to seeing some of my friends join the site eventually. Thanks for the work and answer!

**Reply by Thalita-G** [reply]
@bnewbold Does "non-public content" mean you will be able to change visibility settings per post then? I'd love to see something like Twitter Circles implemented into Bluesky.

**Reply by Rohmaioi** [reply]
Ooh that'd be awesome really. I'd honestly prefer to set an account private (ask to follow and hidden tweets for non-followers) and then giving the option to allow private users to show their comments/replies while being private like Instagram.

**Reply by andrioid** [reply]
Any news on that @bnewbold? Need help or feedback?
